docs/content/preview/releases/ybdb-releases/end-of-life/v2.7.md

```diff
@@ -30,11 +30,11 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
 ### Downloads
 <ul class="nav yb-pills">
   <li>
-   <a  href="https://downloads.yugabyte.com/yugabyte-2.7.2.0-darwin.tar.gz">
+   <a  href="https://software.yugabyte.com/yugabyte-2.7.2.0-darwin.tar.gz">
```

Comment:
I don't think we have a pattern for these old style releases in snarf.  This URL doens't work for me.